src/amo/components/Addon/index.js

```diff
@@ -62,6 +63,8 @@ export class AddonBase extends React.Component {
     RatingManager: PropTypes.element,
     addon: PropTypes.object.isRequired,
     clientApp: PropTypes.string.isRequired,
+    // This prop is passed in by withInstallHelpers()
```

Comment:
👍 documenting stuff like this, especially when it's related to the install helper, is really handy. Thanks!

Reply:
I find the pattern of introducing magical props with an HOC very confusing. I would like to avoid it in the future. I want `withInstallHelpers()` to go away.

Reply:
Ultimately I agree, but this comment is a good stop-gap and will let us refactor it away in the future 😄